# 多个装甲板信息数组
Header header
ArmorInfo[] armors